src/tests/integration/test-data/foundry/test/ConstructorTest.t.sol

```diff
@@ -3,8 +3,34 @@ pragma solidity =0.8.13;
 
 import "forge-std/Test.sol";
 
+contract ImportedContract {
+    uint256 public count;
+
+    constructor() payable {
+        count = 5;
+    }
+
+    function set(uint256 x) public {
```

Comment:
Since we have these functions in place, could we also test them, please, as part of `test_kontrol_cse`? `set`, specifically, revealed the issue we had with the resulting storage not being abstracted.

Reply:
Added `set()`, `add()`, and `count()` to the test.